pkgs/shells/mrsh/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, meson, ninja, pkgconfig, readline }:
+
+stdenv.mkDerivation rec {
+  pname   = "mrsh";
+  version = "ef21854fc9ce172fb1f7f580b19a89d030d67c65";
```

Consider keeping a visible `version` as the date of the snapshot...

```suggestion
  version = "2020-01-08";
```